cumulusci/tasks/bulkdata/step.py

```diff
@@ -220,6 +231,39 @@ def get_results(self):
                 yield from reader
 
 
+class RestApiQueryOperation(BaseQueryOperation):
+    """Operation class for REST API query jobs."""
+
+    def query(self):
+        pass
+
+    def get_results(self):
+        yield from self.sf.query_all_iter(self.soql)
+
+
+class RestApiRetrieveOperation(BaseQueryOperation):
+    def __init__(self, *, sobject, api_options, context, ids, fields):
+        super().__init__(
+            sobject=sobject,
+            operation=DataOperationType.QUERY,
+            api_options=api_options,
+            context=context,
+        )
+        self.ids = ids
+        self.fields = fields
+
+    def query(self):
+        pass
+
+    def get_results(self):
+        for chunk in get_batch_iterator(iter(self.ids), 2000):
+            yield from self.sf.restful(
+                f"composite/sobjects/{self.sobject}",
+                method="POST",
+                json={"ids": chunk, "fields": self.fields},
+            )
```

Comment:
It doesn't look like RestApiRetrieveOperation is used anywhere?

Reply:
I thought I needed it to support Scott's Person Account use case but was mistaken I'll remove it if I don't find a good use for it - it's so nice to be able to pull 2k records at a transaction.